woocommerce/woocommerce
# Change the default currency for Croatia to Euros

Beginning 2023-01-01, Croatia starts using the Euro. For any new stores with a base of operations in Croatia, this should be the default currency from January 1st onwards (though it will still be possible for merchants to manually switch back to HKR if preferred).

For existing stores, no automated change will take place (or, not via the work contained in this PR).

Closes #35872.




### How to test the changes in this Pull Request:

(Updated! We simplified this a lot.)

1. Run the WooCommerce onboarding wizard.
    - You can do this by installing and activating WooCommerce in a brand new site.
    - Or, via the **WooCommerce ▸ Settings** screen, locate the **Help** pulldown and re-run the wizard that way:

<img width="1281" alt="run-onboarding-wizard" src="https://user-images.githubusercontent.com/3594411/207470607-dfd82ba7-de97-43c2-b691-d76c0eabc2d1.png">

2. Within the first onboarding screen, select Croatia as the country/region (most of the other settings can be ignored/you can supply the bare minimum ... and, for expediency, I recommend against installing any extras or switching theme):

<img width="1131" alt="wizard-country-region-croatia" src="https://user-images.githubusercontent.com/3594411/207470827-c3cb4858-20eb-48cc-b66e-bcf74e332303.png">

3. Now navigate to **WooCommerce ▸ Settings ▸ General**:
    - Country/state field should still be `Croatia`.
    - The currency should be `Euro (€)`.

<img width="996" alt="settings-currency-euro" src="https://user-images.githubusercontent.com/3594411/207471589-96d3dd66-9491-4ff6-83a9-242b57c5dce4.png">



### Other information:

- Once/if we are happy with this approach, I will log a follow-up issue so we can come back after the transition period (ie, targeting 7.4 or later) and clean up `i18n/locale-info.php`.
- I considered making the entirety of `i18n/local-info.php` filterable (and we could then make these tweaks via a filter function). Let me know if that's something we prefer to do ... however, since I don't think there has so far been any demand for such a filter I preferred to make a minimal change that does not expand our public API surface any further.
- A little unsure how we want to deal with references to `HKR` in the future, post-transition (when it ceases to be a valid currency), but we can also handle that at some later point/via the follow-up issue.

### FOR PR REVIEWER ONLY:
